src/components/common/Modal/index.tsx

```diff
@@ -7,9 +7,11 @@ interface ModalProps {
   open: boolean;
   onClose: () => void;
   children?: ReactNode;
+  /** Anchor content to the bottom of the screen. Primarily for mobile modals. */
```

Comment:
It looks like bottomSheet doesn't apply unless you're on mobile now. Maybe you can update the comment to "Only applies for mobile modals"